aws-janitor/resources/route53.go

```diff
@@ -96,7 +96,9 @@ func route53ResourceRecordSetsForZone(opts Options, logger logrus.FieldLogger, s
 			}
 			logger.Warningf("%s: deleting %T: %s", o.ARN(), rrs, *rrs.Name)
```

Comment:
We should move the Skip check so it doesn't log this if it's not actually going to delete the record.